tensorflow/lite/python/lite_v2_test.py

```diff
@@ -1192,5 +1192,81 @@ def model(in_tensor):
         str(error.exception))
 
 
+class AffineOpThenMulFusionTest(lite_v2_test_util.ModelTest):
```

Comment:
func1/func2/... are very general names. Please use more descriptive names to what it is testing, and add comments in needed parts please.

Thanks

Reply:
I refactored the tests to make them clearer and expanded the test coverage.